### DISCUSSION TITLE
Best approach for configuration in Lua

### DISCUSSION THREAD

**Discussion by Milo123459** [mlua-rs/mlua#221] [Q&A] [answered]
Hey!

I'm just curious on what the best approach for creating some sort of configuration object is via mlua. It should be accessible and editable via the Lua code and readable on the Rust side, for example:
```lua
Config {
   Routes = {...
}
```
and I could then access that on the rust side via
```rs
config.routes
```
or something like that.

Not sure what the best approach is, please let me know!

**Comment by khvzak** [comment, marked as answer]
have you tried to use serde for that? just define struct in rust and deserialize lua object into it.
